src/Algebra/Graph/HigherKinded/Class.hs

```diff
@@ -500,7 +508,7 @@ deBruijn :: Graph g => Int -> [a] -> g [a]
 deBruijn 0   _        = edge [] []
 deBruijn len alphabet = skeleton >>= expand
   where
-    overlaps = mapM (const alphabet) [2..len]
+    overlaps = traverse (const alphabet) [2..len]
```

Comment:
@phadej I'm confused about this change. This doesn't seem to be for the sake of compatibility -- I just tried to undo it and all CI instances passed. Could you clarify? Happy to do it, but I'd like to understand the reason.

Reply:
Furthermore, there are few more instances of `mapM` in the code, so if we change this one, we should probably changes others too.